### PR TITLE
adding full brain rotation 

### PR DESCRIPTION
**Pull request by silviafan**
When view='360', the specified frames will go from L view around and excluding L view.

### CODE REVIEW

**Comment by wiheto** [comment]
Looks good. Just fix the one trailing whitespace error that the checks identified: https://app.codacy.com/gh/wiheto/netplotbrain/pullRequest?prid=10003511 it should be good to go.

File: netplotbrain/utils/plot_utils.py [hunk 1]
```diff
@@ -64,17 +64,23 @@ def _get_view(views='L', frames=1, arrowaxis='auto'):
...
         vy = []
         for view in views:
             vxtmp, vytmp = view_defaults[view]
             vx.append(vxtmp)
             vy.append(vytmp)
     # Otherwise rotate figure based on frames iput
-    else:
-        v1x, v1y = view_defaults[fromview]
-        v2x, v2y = view_defaults[toview]
+    else:
```
Comment (wiheto) [inline netplotbrain/utils/plot_utils.py:75]: trailing whiitespace here

Comment (silviafan) [reply netplotbrain/utils/plot_utils.py]: Right, done! :)

**Comment by wiheto** [comment]
Great LGTM

**Pull request merged**